### DISCUSSION TITLE
Newsletter scenario

### DISCUSSION THREAD

**Discussion by bdavidxyz** [janko/rodauth-rails#85] [Q&A] [answered]
Just wondering, I'm opening a discussion because I'm not able to find an answer after reading the feature list.

Is it possible to have "subscribe to newsletter" feature with rodauth-rails ? 

Let's say our Rails app has a blog. At the end of each blog article : there is a form to subscribe to a newsletter.

Problem : most people that read the blog are not connected users. It's just  interested visitors, that want to subscribe to the newsletter, but not yet signup to the app.

So we have here "unauthenticable users". They must receive double opt-in confirmation (to prevent spam), they must receive newsletters after this confirmation, so they are actually users of our Rails app. But at this point, they can't authenticate to the app.

An option of course is to use external services like ConvertKit. 

However, is there a way to build this "newsletter scenario" with rodauth-rails ?

**Comment by janko** [comment, marked as answer]
It sounds to me like the newsletter feature is orthogonal to authentication, and that the only point of contact is probably pre-filling the email field with the email address of the currently logged in account. It doesn't seem like this is something that should be hooked onto authentication.

**Reply by bdavidxyz** [reply]
That's also what I was thinking the past few days. I should here split the marketing notions of "lead" and "customer". Two different things. You underlined the tricky part : what if the "lead" become a "customer" ? This point is outside the scope of Rodauth, definetely.   Thanks for the insight !
